### ISSUE TITLE
Voice Search not working from Home Screen

### ISSUE THREAD

**Issue by Natureboy251** [koying/SPMC#204]
I have the Nvidia Sheild 16gb.  I have enabled the web interface in SPMC using default settings.  I ensure that SPMC is running before returning to the home screen.

I can search within SPMC using the repurposed IP in the soft keyboard, but for the life of me I cannot get SPMC search results to pop up from the Android TV Home Screen.

I have googled this issue to death and am stuck :(

I downloaded SPMC Version 16.2 from the Google Play Store.

Oh, and I am using the Nvidia Sheild Remote, not the game controller.

**Comment by beschoenen** [comment]
Did you enable SPMC for voice search in the Android settings?
I believe it's under "Home"

**Comment by Natureboy251** [comment]
SPMC isn't among the apps listed there :(

**Comment by Semskiy** [comment]
Hello, i got the same problem. 
I dont have any news from "spmc" on home screen and i never get search results from it, apart from clicking IP like the OP.
In "Settings", "Search", "Searchable apps" SPMC is included.

I tried changing Languages, System and Voice. Restarting the system many times. Can't do anything to make it work! Pls help!
The voice search is what attracted me to get SPMC in the first place instead of Kodi. At least the TTS from pressing "IP" works, love it and find it amazing!
- Shield Android TV 16GB Version 6.0 (Everything updated SPMC too)

**Comment by koying** [comment]
Did you enable the SPMC Web server?

**Comment by Natureboy251** [comment]
Yes I enabled the web server in SPMC settings. Maybe the Google Play Store
version sucks and I should have side loaded the apk instead.

**Comment by Semskiy** [comment]
For me, when i opened up SPMC, web server was already enabled. I tried disabling and enabling again and nothing works. Edit: Restored to default as well.
I hope its not because of my geo location (Russia) 

Feel left out because everybody has this feature apart from OP and me =))

**Comment by Semskiy** [comment]
BTW i also dont have any suggestions from SPMC on the Android TV home screen if that helps.

**Comment by Semskiy** [comment]
Installed the newest version of SPMC 16.2.1 but still nothing works, SPMC definitely running in background, tried searching within the app and from shield tv home screen. No results.
- The search function searches through "Video Addons"? Or only through the films i have downloaded and have in the SPMC library or something???

I'm sorry with all these dumb question, i really did try searching answers through google but cant find answers.

**Comment by Katfish69uk** [comment]
I would like to know the solution to this too as i have been trying for 4 days now to get the voice search from the Shield home screen to work.

Voice search works with in SPMC.

But nothing when searching from the home screen of the Nvidia Shield  nor any recommendations on the home screen.

Webserver enabled with defult settings port 8080 username kodi password blank web interface default webinterface.

SPMC show up in Search in settings   search of the Nvidia Shield  but SPMC dose not show up in Settings   home screen   recommendations row.

Hopefully someone can do a checklist to trouble shoot and help us out to hooefully get this simple operation fixed.

**Comment by koying** [comment]
You actually have a library, right?
Does not work with addons nor with non-library files.

**Comment by Katfish69uk** [comment]
How do you do a libary from an addon so if self updates?
